### PR TITLE
DW-235:Push notification POC

### PR DESCRIPTION
**Pull request by cbernat**
### Web push notifications without BE
Following google's [tutorial](https://developers.google.com/web/fundamentals/codelabs/push-notifications), a mini test in ui library yet without back end. 

image

@andresmoschini @fcoronelmakingsense perhaps you are interested in this.

To test notifications locally you can use this:
https://cdn.fromdoppler.com/doppler-ui-library/build754/notification-poc.html

This works in chrome and firefox, Edge it seems not.

- [X] Check if there is a new color.
- [x] Create the color variable in the _colors.scss file.
- [x] Follow the color nomenclature.
- [x] Create the color class with its variable.
- [x] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Comment by cbernat** [comment]
test this please.

File: assets/templates/notification-component.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>UI library Doppler - Breadcrumb component</title>
```
Comment (GusBaamonde) [inline assets/templates/notification-component.html]: maybe Push Notifications POC?

Comment (cbernat) [reply assets/templates/notification-component.html]: Yess It's copy/paste curse 😜

**Comment by cbernat** [comment]
test this please.

File: assets/js/webPushMain.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/*** Web push test ****/
+var applicationServerPublicKey = 'BFtBVunx6Lg_4ziA6b_Oe-9iqQetudiBzkukb1UOOWItZlbdPXCnMLjTfM6gnbmzrusaGlifwWWaFRWnef_H40Y';
+// Public Key
+// BFtBVunx6Lg_4ziA6b_Oe-9iqQetudiBzkukb1UOOWItZlbdPXCnMLjTfM6gnbmzrusaGlifwWWaFRWnef_H40Y
+// Private Key
+// F1eZLlbqpH5d8nHetf6xzzBnlgq8WbFlQ2A9jEl9X4E
```
Comment (andresmoschini) [inline assets/js/webPushMain.js]: Is it fine including this in a public repository? (if not, please consider revoke this key soon)

Comment (cbernat) [reply assets/js/webPushMain.js]: This is only to easily test, not sensitive information at all. In the real life yes we would not put this information here.

**Comment by cbernat** [comment]
test this please.

**Pull request merged**